app/src/main/java/se/cygni/paintbot/game/WorldUpdater.java

```diff
@@ -255,44 +260,10 @@ private int getNextPosition(Map.Entry<String, Action> updateEntry, WorldState ws
         }
     }
 
-
     private void syncPoints(WorldState ws) {
         playerManager.toSet().forEach(player -> {
             if (player.isAlive())
                 ws.getCharacterById(player.getPlayerId()).setPoints(player.getTotalPoints());
         });
     }
-
-    private void notifyCharacterStunned(
```

Comment:
I guess this wasn´t used?

Reply:
Didn't seem like it. Also found a bunch of other stuff later on that was marked as not used 🤷🏻‍♀️